ui/src/assets/i18n/de.json

```diff
@@ -0,0 +1,1082 @@
+      "runtime": "Laufzeit",
+      "tasks": "Aufgaben",
+      "tasksJobs": "Aufgaben / Jobs",
+      "taskExecution": "Aufgaben-Ausführungen",
+      "jobExecution": "Job-Ausführungen",
+      "schedules": "Zeitpläne",
+      "manage": "Verwalten",
+      "auditRecords": "Audit-Einträge",
+      "tools": "Tools"
+    },
+    "quicksearch": {
+      "input": "Such nach Schlüsselworten...",
+      "searching": "Suche nach Ergebnissen...",
+      "application": "Anwendung(en)",
+      "stream": "Stream(s)",
+      "task": "Aufgabe(n)",
+      "noResult": "Es gibt keine Suchergebnisse für"
+    }
+  },
+  "applications": {
+    "main": {
+      "title": "Anwendungen",
+      "addApplications": "Anwendung(en) hinzufügen",
+      "unregisterApplications": "Anwendung(en) deregistrieren",
+      "name": "Name",
+      "type": "Typ",
+      "version": "Version",
+      "uri": "URI",
+      "metadataURI": "Metainformationen URI",
+      "manageVersion": "Version verwalten",
+      "unregister": "Deregistrieren",
+      "noResult": "Keine Ergebnisse gefunden",
+      "applicationsPerPage": "Anwendungen pro Seite",
+      "application": "Anwendung",
+      "details": "Details",
+      "pagination": "{{ first }} - {{ last }} von <span class=\"pagination-total\">{{ total }}</span> Anwendungen"
+    },
+    "add": {
+      "registerOneOrMore": "Registriere eine oder mehr Anwendungen",
+      "importFromHttp": "Anwendung von einer HTTP URI Zielkoordinate importieren",
+      "importFromFile": "Anwendung von einer Properties-Datei importieren",
+      "importFromSpringDataFlow": "Anwendung von dataflow.spring.io-Starter importieren.",
+      "websiteStarters": {
+        "usePredefined": "Benutze vordefinierte Anwendungs-URIs (für Maven oder Docker) für alle  out-of-the-box Stream/Aufgaben/Batch-Anwendungen.",
+        "startersForKafkaMaven": "Stream Anwendungen-Starter für Kafka/Maven",
+        "startersForKafkaDocker": "Stream Anwendungen-Starter für Kafka/Docker",
+        "startersForRabbitMQMaven": "Stream Anwendungen-Starter für RabbitMQ/Maven",
+        "startersForRabbitMQDocker": "Stream Anwendungen-Starter für RabbitMQ/Docker",
+        "startersForMaven": "Aufgaben Anwendungen-Starter für Maven",
+        "startersForDocker": "Aufgaben Anwendungen-Starter für Docker",
+        "forceImport": "Forcieren, Anwendungen werden importiert und installiert auch wenn sie bereits existieren aber nicht verwendet werden.",
+        "importApplications": "Importiere Anwendung(en)",
+        "importingApplications": "Importiere gerade Anwendung(en)",
+        "message": {
+          "successTitle": "Importiere Starter",
+          "successContent": "Anwendung(en) importiert.",
+          "noStarterTitle": "Kein Starter ausgewählt",
+          "noStarterContent": "Bitte wählen Sie ein Starter Packet aus."
+        }
+      },
+      "props": {
+        "desc": "Geben Sie eine Liste von Eigenschaften in das hier drunter liegende Textfeld ein.",
```

Comment:
Geben Sie eine Liste von Eigenschaften in das darunter liegende Textfeld ein.

Reply:
Much better. Thanks!